src/db/migrations/migration2.js

```diff
@@ -0,0 +1,33 @@
+const { getLogger } = require('../../utils/logger');
+const { map } = require('lodash');
+
+async function addTypeField(db) {
+  try {
+    // await db.update({ type: { $exists: false }, token: 'QTUM' }, { $set: { type: 'BET' } }, { multi: true });
+    // await db.update({ type: { $exists: false }, token: 'QTUM' }, { $set: { type: 'BET' } }, { multi: true });
+    const votes = await db.Votes.find({ type: { $exists: false } });
+    map(votes, async (vote) => {
+      if (vote.token === 'QTUM') {
+        await db.Votes.update({ txid: vote.txid }, { $set: { type: 'BET' } });
+      } else if (vote.token === 'BOT') {
+        const oracle = await db.Oracles.find({ address: vote.oracleAddress });
+        if (oracle[0].token === 'QTUM') await db.Votes.update({ txid: vote.txid }, { $set: { type: 'RESULT_SET' } });
+        else if (oracle[0].token === 'BOT') await db.Votes.update({ txid: vote.txid }, { $set: { type: 'VOTE' } });
+      }
+    });
+  } catch (err) {
+    getLogger().error(`DB update Error: ${err.message}`);
+    throw Error(`DB update Error: ${err.message}`);
```

Comment:
lets be more specific for the error message. something like `Migration 2 error: ${err.message}`